contract/src/start-game1-proposal.js

```diff
@@ -117,6 +136,13 @@ const gameManifest = {
 };
 harden(gameManifest);
 
+/**
+ * Get the manifest for the game1 module.
+ *
+ * @param {Object} restoreRef - Object containing a reference restoration function.
```

Comment:
That says that `restoreRef` is the whole parameter, when the code says that `restoreRef` is a property of the 1st parameters.